examples/paywall-tester/build.gradle

```diff
@@ -45,17 +46,11 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
     kotlinOptions {
-        jvmTarget = '1.8'
-    }
-    buildFeatures {
-        compose true
+        jvmTarget = '19'
```

Comment:
This seems quite high and doesn't correspond to the `targetCompatibility` set above. Is this needed for anything? (Also in `api-tester/build.gradle`.)

Reply:
hmm, good point, I saw it in a snippet in https://developer.android.com/jetpack/androidx/releases/compose-kotlin#declaring_dependencies 

what do you think? if I understand correctly, this is the version of the jvm our code should be compatible with? So maybe we should keep it as 1.8?

Reply:
Yeah that's how I understand it as well. [Java 17 is the latest supported version on Android 14](https://developer.android.com/build/jdks#compileSdk), and [desugaring is only available for Java 11 & 8](https://developer.android.com/studio/write/java8-support#library-desugaring). I have no idea why that snippet sets it to 19. 😕  
  
I think keeping it at 1.8 (8) is the safest indeed, unless we need any of the new Java features.